docs/mp/testing-ng.adoc

```diff
@@ -119,23 +122,71 @@ In current example Helidon container will be launched prior test. The _Bean Disc
 [source,java]
 .Code sample
 ----
-@HelidonTest
-@DisableDiscovery
-@AddBean(MyBean.class)
-@AddExtension(ConfigCdiExtension.class)
-@AddConfig(key = "app.greeting", value = "TestHello")
+@HelidonTest    <1>
+@DisableDiscovery   <2>
+@AddBean(MyBean.class)  <3>
+@AddExtension(ConfigCdiExtension.class) <4>
+@AddConfig(key = "app.greeting", value = "TestHello")   <5>
 class TestExample {
     @Inject
-    private MyBean myBean;
+    private MyBean myBean;  <6>
 
     @Test
-    void testGreeting() {
+    void testGreeting() {   <7>
         assertThat(myBean, notNullValue());
         assertThat(myBean.greeting(), is("TestHello"));
     }
 }
 ----
+<1> Start the Helidon container.
+<2> Set disabled Bean Discovery for the current test class.
+<3> Add `MyBean` to current context.
+<4> Add a configuration CDI extension to the current test.
+<5> Add configuration properties.
+<6> Inject `MyBean` as it is available in the CDI context.
+<7> Run rests.
+
+To test `@RequestScoped` bean with JaxRs support:
+
+[source,java]
+.Test `RequestScoped` bean
+----
+
+@HelidonTest    <1>
+@DisableDiscovery <2>
+@AddExtension(ServerCdiExtension.class)
+@AddExtension(JaxRsCdiExtension.class)
+@AddExtension(CdiComponentProvider.class)
 
+@AddJaxRs <3>
```

Comment:
`@AddJaxRs` should replace the need for manually adding extensions above
```suggestion
@AddJaxRs <3>
```

Reply:
Yep, fixed.